processing/src/main/java/org/apache/carbondata/processing/datatypes/GenericDataType.java

```diff
@@ -71,6 +71,8 @@ void writeByteArray(T input, DataOutputStream dataOutputStream)
    */
   void setSurrogateIndex(int surrIndex);
 
+  Boolean getIsColumnDictionary();
```

Comment:
change this to boolean isDictionaryEncoded()

Reply:
Done

Reply:
Done